perun-base/src/main/java/cz/metacentrum/perun/core/api/Candidate.java

```diff
@@ -19,6 +20,9 @@
  */
 public class Candidate extends User {
 
+	private String NS_USER_ATTR_CORE = "urn:perun:user:attribute-def:core:";
+	private String NS_USER_ATTR = "urn:perun:user:attribute-def:";
```

Comment:
Should be also ``static`` and ``final``.

Reply:
Fixed.